rfcs/text/0018_timeslider.md

```diff
@@ -0,0 +1,217 @@
+- Start Date: 2020-04-26
+- RFC PR: (leave this empty)
+- Kibana Issue: (leave this empty)
+
+---
+- [1. Summary](#1-summary)
+- [2. Detailed design](#2-detailed-design)
+- [3. Unresolved questions](#3-unresolved-questions)
+
+# 1. Summary
+
+A timeslider is a UI component that allows users to intuitively navigate through time-based data.
+
+This RFC proposes adding a timeslider control to the Maps application. 
+
+It proposes a two phased for roll-out in the Maps application. The [design proposal](#2-detailed-design) focuses on the first phase.
+
+Since the a timeslider UI is relevant for other Kibana apps, the implementation should be portable. We propose to implement the control as a React-component 
```

Comment:
Remove "a" in "Since the a timeslider UI is relevant for other Kibana apps," so sentence becomes "Since the timeslider UI is relevant for other Kibana apps,"